tests/data/profile_stability/rhel8/stig.profile

```diff
@@ -21,7 +21,6 @@ description: 'This profile contains configuration checks that align to the
 
     - Red Hat Containers with a Red Hat Enterprise Linux 8 image'
 documentation_complete: true
-extends: ospp
```

Comment:
Is this intended?

Reply:
Yes, reference compiled profiles were made as copies of past compiled profiles that preserved the `extends:`, which was a bug. Once the profile is compiled, it is self-contained.